packages/assemble-lite/helper/template-helper.js

```diff
@@ -0,0 +1,29 @@
+
+const { basename } = require("path");
+
+const { getPaths, asyncReadFile } = require("./io-helper");
+
+const loadTemplates = async templates => {
+  const templatePaths = await getPaths(templates);
+  const templMap = new Map();
+  await Promise.all(templatePaths.map(async path => {
+    const filename = basename(path, ".hbs");
+    const markup = await asyncReadFile(path);
+
+    templMap.set(filename, markup);
+  }));
+  return templMap;
+};
+
+
+const applyTemplate = (templMap, templateName, page) => {
+  if (!templMap.has(templateName)) return page;
+
+  const template = templMap.get(templateName);
+  return template.replace(/{%\s*body\s*%}/g, page);
```

Comment:
I'm not sure but could'nt you just use a {{{body}}} block in your hbs files instead of replacing it here manually ?

Reply:
I tried do do it like this, but it had some side-effects.